iXBRLViewerPlugin/viewer/src/js/util.js

```diff
@@ -13,25 +13,38 @@
 // limitations under the License.
 
 import dateFormat from "dateformat"
+import moment from "moment";
 
-export function isodateToHuman(d, adjust) {
-    if (d.getUTCHours() + d.getUTCMinutes() + d.getUTCSeconds() == 0) { 
+/* 
+ * Takes a moment.js oject and converts it to a human readable date, or date
+ * and time if the time component is not midnight.  Adjust specifies that a
+ * date (but not a date time) should be shown as the day before.  This is to
+ * satisfy the convention of describing durations using inclusive dates.
+ */
+export function momentToHuman(d, adjust) {
+    if (d.hours() + d.minutes() + d.seconds() == 0) { 
         if (adjust) {
-            d.setDate(d.getDate() - 1);
+            d = d.clone().subtract(1, 'day');
         }
-        return dateFormat(d,"d mmm yyyy");
+        return d.format('D MMM Y');
     }
     else {
```

Comment:
You can remove this else statement and just use the return from line 32.

Reply:
Done in 470bd1b